app/views/billboards/index.html.erb

```diff
@@ -0,0 +1,62 @@
+<% provide :head_tags do %>
+<meta name='turbolinks-visit-control' content='reload'>
+    <script>
+
+
+    document.addEventListener("DOMContentLoaded", function(){
+    	initMap();
+    });
+
+
+
+
+    function initMap() {
+    	var marker;
+    	var newCoord;
+    	var center_lat = 0;
+    	var center_lng = 0;
+    	var count = 0
+    var myCoords = new google.maps.LatLng(53.9, 27.559);
+    var mapOptions = {
+    center: myCoords,
+    zoom: 14
+    };
+
+    var map = new google.maps.Map(document.getElementById('map'), mapOptions);
+
+    <% @billboards.each do |billboard| %>
```

Comment:
This is something weird.
1. You can render `var billboards = <%= @billboards.to_json %>` and just use here. This mix of JS and ERB is hard to maintain.
2. `count++` also should go away as it could be replaced with `billboards.length`.
3. `center_lat` and `center_lng` don't match their names. should be something like `average_lat/average_lng`, and you could get them by running `reduce` on `billboards`